apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/LabelProviders/EnvironmentNameLogEntryLabelProvider.cs

```diff
@@ -38,7 +38,7 @@ public EnvironmentNameLogEntryLabelProvider(IHostingEnvironment hostingEnvironme
         /// <inheritdoc/>
         public void Invoke(Dictionary<string, string> labels)
         {
-            labels["aspnetcore_environment"] = _hostingEnvironment.EnvironmentName;
+            labels["aspnetcore_environment"] = _hostingEnvironment.EnvironmentName ?? "N/A";
```

Comment:
And also, add a test for when the `EnvironmentName` is null to check that there's no label added.